osu.Framework/Platform/Storage.cs

```diff
@@ -123,6 +123,11 @@ public virtual Storage GetStorageForDirectory(string path)
         /// <summary>
         /// Opens a native file browser window to the root path of this storage.
         /// </summary>
-        public abstract void OpenInNativeExplorer();
+        public virtual void OpenInNativeExplorer() => OpenPathInNativeExplorer(string.Empty);
```

Comment:
Doesn't need to be virtual IMO, `OpenPathInNativeExplorer` should be enough to handle that.
```suggestion
        public void OpenInNativeExplorer() => OpenPathInNativeExplorer(string.Empty);
```

Reply:
See the osu!-side PR.